octobot_trading/personal_data/orders/order_util.py

```diff
@@ -435,3 +437,55 @@ async def get_order_size_portfolio_percent(exchange_manager, order_amount, side,
 
 def generate_order_id():
     return str(uuid.uuid4())
+
+
+async def update_from_order_storage(order, exchange_manager, pending_groups):
+    # only real orders can be updated by stored orders
+    if exchange_manager.is_trader_simulated or exchange_manager.is_backtesting \
+            or not exchange_manager.storage_manager.orders_storage:
+        return
+    order_details = await exchange_manager.storage_manager.orders_storage.get_startup_order_details(order.order_id)
+    if order_details:
+        await _update_from_order_details(order, order_details, exchange_manager, pending_groups)
```

Comment:
Should we return here?

Reply:
the is no return to the function, i'm not sure i understand what you mean